packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -263,6 +268,7 @@ const start = async (zcf, privateArgs) => {
     zcf,
     isSecondary,
     secondaryBrandToLiquidityMint,
+    secondaryBrandToLiquidityIssuerPromise,
```

Comment:
it's not called out as promises in the callee. Why does this need to get provided as an argument instead of allocaed in the callee?

Reply:
You're right that I could (and maybe should) have put it in makeAddIssuer, but we'll be moving to durability shortly, and at that point, addIssuer will need to get the collection from its caller anyway.

Reply:
I would not expect it to expose its internal state as opposed to simply managing it around baggage, but I agree it will get sufficiently shuffled that this doesn't matter,